params/superchain.go

```diff
@@ -32,7 +32,7 @@ var KromaChainConfigs = map[uint64]*KromaChainConfig{
 	KromaDevnetChainID: {
 		CanyonTime:   uint64ptr(1707292800),
 		EcotoneTime:  uint64ptr(1712908800),
-		KromaMPTTime: uint64ptr(1735189200),
+		KromaMPTTime: uint64ptr(1735534800),
```

Comment:
I think the time should be located at the point after the upgrade to v0.6.0, maybe Tuesday 2pm or Thursday 2pm.

Reply:
Okay then I think Thursday seems like a good option.